lib/rugged_adapter/git_layer_rugged.rb

```diff
@@ -651,12 +651,25 @@ def log(commit = Gollum::Git.default_ref_for_repo(@repo), path = nil, options =
         git.log(commit, path, **options)
       end
 
-      def lstree(sha, options = {})
-        results = []
-        @repo.lookup(sha).tree.walk(:postorder) do |root, entry|
-          results << ::Gollum::Git::Tree.tree_entry_from_rugged_hash(entry, root)
+      def lstree(sha, path, options = {})
```

Comment:
In RJGit adapter, the `path` argument in `lstree` has a default of `nil`. Should we do the same here?